### DISCUSSION TITLE
Why have to relogin at times?

### DISCUSSION THREAD

**Discussion by juxeii** [photoprism/photoprism#2011] [Q&A] [answered]
Is there a setting which determines if you need to login again after some time?
I experience that photoprism (on Windows and Android browsers) needs the username and password after some sessions?!
Why is that and what can I do about it?

**Comment by lastzero** [comment, marked as answer]
It prevents...

1. indefinite access to your instance in case someone gets their hands on your credentials or browsing session (for example, if you share it with friends)
2. the backend from accumulating an unlimited number of sessions that may never be used again

Of course, users may have different preferences, which is why we are currently reworking the authentication options and settings.
